docs/GettingStarted.md

```diff
@@ -65,7 +65,8 @@ of the Datadog tracer, you can override the following defaults:
       debug: false,
       trace_agent_hostname: 'localhost',
       trace_agent_port: 8126,
-      env: Rails.env,
+      use_rails_env: false,
```

Comment:
what I'm worried about that, is we're polluting a lot Rails settings to provide "almost any kind of behavior" with flags rather than be explicit (i.e.: if you want to use the Rails env, just set `env: Rails.env`)